package.json

```diff
@@ -10,7 +10,7 @@
     "prepare": "husky install",
     "build": "nx run-many --target=build",
     "lint": "nx run-many --target=lint",
-    "test": "nx run-many --target=test -- --watchAll=false --silent=true",
+    "test": "nx run-many --target=test:ci",
```

Comment:
Create `test:ci` jobs explicitly for running in parallel (vitest doesn't use `watchAll` so the interactive test runner would be launched if running this script locally)